common/shlibs

```diff
@@ -498,6 +498,8 @@ libfsimage.so.1.0 xen-libs-4.10.0_1
 libxenstat.so.0 xen-libs-4.10.0_1
 libfam.so.0 gamin-0.1.10_1
 libgamin-1.so.0 gamin-0.1.10_1
+libgammaray_core-qt5_15-x86_64.so.2.11.1 GammaRay-2.11.1_1
+libgammaray_common-qt5_15-x86_64.so.2.11.1 GammaRay-2.11.1_1
```

Comment:
The `-qt5_15-x86_64` part should be removed or `shlibs` will be off on other architectures. To make the build produce sane file names, `CMakeLists.txt` will have to be fixed, like in the `shiboken2` package ([patch](https://github.com/void-linux/void-packages/blob/master/srcpkgs/shiboken2/patches/no-python-arch-suffix.patch)).